release_tester/arangodb/async_client.py

```diff
@@ -75,9 +75,9 @@ def run_arango_tool_monitored(
         self,
         executeable,
         more_args,
-        timeout,
-        result_line,
-        verbose,
+        timeout=300,
```

Comment:
a launched program sitting dead in the water for 5 minutes? I'd lay 1 minute is more than enough.